src/actinia_core/core/common/process_chain.py

```diff
@@ -1210,6 +1267,15 @@ def _add_grass_module_input_parameter_to_list(
                         file_id
                     ] = self.generate_temp_file_path()
                 param = "%s=%s" % (param, self.temporary_pc_files[file_id])
+            elif "::" in value and value.split("::")[1] in [
+                "stdout",
+                "stderr",
+            ]:
+                id = module_descr["id"]
```

Comment:
`id` is unused? Should it be an argument to `self._create_param_stdin_process()` in the next line?